### ISSUE TITLE
show contributors

### ISSUE THREAD

**Issue by freeman-lab** [freeman-lab/minidocs#31]
It might be cool if we had an extra option for showing contributors to a project, in the style of the [`stack.gl`](http://stack.gl/) site.

It'd work something like this: you specify one or more GitHub handles as an extra option, and we append a collection of small images (that link to GitHub profile pages) at the bottom of the sidebar on the left. I can put together a mock and add it to this issue.

Thoughts?

**Comment by yoshuawuyts** [comment]
Ooh, that'd be cool! - perhaps show contributers on a per-document basis? e.g. for large docs it might be useful to see who worked on what / who wrote what

**Comment by freeman-lab** [comment]
Ah, even better! And that should be easy with our API for `contents`, because each doc can either be a single markdown file or an object, like this:

```js
'modules': {
    'welcome': 'welcome.md',
    'cool-package': {
      file: 'readme.md',
      link: 'http://github.com/yoshuawuyts/cool-package',
      contributors: ['yoshuawuyts', 'sethdvincent']
    }
  } 
```

And here's a mockup (including a tweaked look for the "source" link)

![screen shot 2016-06-21 at 10 48 11 am](https://cloud.githubusercontent.com/assets/3387500/16233935/f43c82bc-379d-11e6-943b-b20d0544b17b.png)

Reasonable to assume any username refers to GitHub?

**Comment by yoshuawuyts** [comment]
> Reasonable to assume any username refers to GitHub?

Yeah, I def think so

**Comment by freeman-lab** [comment]
Cool, I'll roll up a PR for this!

**Comment by sethvincent** [comment]
🎉 closed by #35
